src/test/java/org/apache/commons/validator/routines/IBANValidatorTest.java

```diff
@@ -55,7 +55,7 @@
 /**
  * Tests {@link IBANValidator}.
  */
-public class IBANValidatorTest {
+class IBANValidatorTest {
```

Comment:
Not connected with change

Reply:
I agree. Please keep your changes to a minimum.